src/runtime/pkg/oci/utils_test.go

```diff
@@ -821,22 +821,22 @@ func TestRegexpContains(t *testing.T) {
 
 	//nolint: govet
 	type testData struct {
-		regexps  []string
 		toMatch  string
+		regexps  []string
 		expected bool
 	}
 
 	data := []testData{
-		{[]string{}, "", false},
-		{[]string{}, "nonempty", false},
-		{[]string{"simple"}, "simple", true},
-		{[]string{"simple"}, "some_simple_text", true},
-		{[]string{"simple"}, "simp", false},
-		{[]string{"one", "two"}, "one", true},
-		{[]string{"one", "two"}, "two", true},
-		{[]string{"o*"}, "oooo", true},
-		{[]string{"o*"}, "oooa", true},
-		{[]string{"^o*$"}, "oooa", false},
+		{regexps: []string{}, toMatch: "", expected: false},
```

Comment:
This series of changes seems quite welcome, but belongs to another commit.

Reply:
This is/was due to fieldalignment, it is disable now and I can revert that.